src/Annotation/Provider.php

```diff
@@ -14,8 +14,20 @@ final class Provider implements Annotation
 {
     /**
      * Optionnal prefix for provider fields.
-     *
-     * @var string
      */
     public string $prefix;
+
+    /**
+     * The default target types to attach the provider queries to.
+     *
+     * @var array<string>
+     */
+    public array $targetTypeQuery;
+
+    /**
+     * The default target types to attach the provider mutations to.
+     *
+     * @var array<string>
+     */
+    public array $targetTypeMutation;
```

Comment:
Wouldn't it be better to name the property `$targetMutationTypes`?

Reply:
Done